docs/modules/extensions/nav.adoc

```diff
@@ -1,11 +1,14 @@
 * xref:index.adoc[]
```

Comment:
The document processors should be grouped under "Document Processors" and the syntax processors grouped under "Syntax Processors". Otherwise, it just looks like a laundry list of extension points.

Reply:
They are arranged currently in order of invocation.  I've tried adding the type to the page titles, WDYT?  Perhaps having a heading, "processors in order of invocation" would be useful?  If you still want a type/alphabetic arrangement, OK.